src/current/_includes/cockroachcloud/quickstart/create-free-trial-standard-cluster.md

```diff
@@ -1,18 +1,14 @@
-{% comment %}
-After the free trial details are finalized, add a link to free trial page to check for free trial eligibility.
-{% endcomment %}
-
-1. Create a [CockroachDB {{ site.data.products.cloud }}](https://cockroachlabs.cloud/signup?referralId={{page.referral_id}}) account and log in.
-1. On the **Clusters** page, click **Create cluster**.
+1. [Create a CockroachDB {{ site.data.products.cloud }} account]({% link cockroachcloud/create-an-account.md %}). If this is your first CockroachDB Cloud organization, it will be credited with $400 in [free trial credits]({% link cockroachcloud/free-trial.md %}) to get you started.
+1. On the **Get Started** page, click **Create cluster**.
 1. On the **Select a plan** page, select **Standard**.
-1. On the **Cloud & Regions** page, select a cloud provider (GCP or AWS) in the **Cloud provider** section.
+1. On the **Cloud & Regions** page, select a cloud provider (GCP or AWS).
 1. In the **Regions** section, select a region for the cluster. Refer to [CockroachDB {{ site.data.products.cloud }} Regions]({% link cockroachcloud/regions.md %}) for the regions where CockroachDB {{ site.data.products.standard }} clusters can be deployed. To create a multi-region cluster, click **Add region** and select additional regions.
 1. Click **Next: Capacity**.
-1. On the **Capacity** page, keep the **Provisioned capacity** at the default value of 1000 RUs/second.
+1. On the **Capacity** page, keep the [**Provisioned capacity**]({% link cockroachcloud/plan-your-cluster.md %}) at the default value of 2 vCPUs.
 
     Click **Next: Finalize**.
 
-1. On the **Finalize** page, name your cluster. If applicable, the 30-day trial code is pre-applied to your cluster.
+1. On the **Finalize** page, name your cluster. If an active free trial is listed in the right pane, you will not need to add a payment method, though you will need to do this [by the end of the trial]({% link cockroachcloud/free-trial.md %}#add-payment-methods) to further maintain your organization and its clusters.
```

Comment:
```suggestion
1. On the **Finalize** page, optionally name your cluster or choose the randomly-assigned name. If an active free trial is listed in the right pane, you will not need to add a payment method. Before the  [end of the trial]({% link cockroachcloud/free-trial.md %}#add-payment-methods), you must add a payment method to keep using the cluster..
```
I do not believe you need a payment method to keep the organization. Is it worth noting that you will be reminded about this near the end of the trial?